controllers/openstackdataplane_controller.go

```diff
@@ -54,6 +54,7 @@ type OpenStackDataPlaneReconciler struct {
 //+kubebuilder:rbac:groups=dataplane.openstack.org,resources=openstackdataplanes/status,verbs=get;update;patch
 //+kubebuilder:rbac:groups=dataplane.openstack.org,resources=openstackdataplanes/finalizers,verbs=update
 //+kubebuilder:rbac:groups=dataplane.openstack.org,resources=openstackdataplaneservices,verbs=get;list;watch
+// +kubebuilder:rbac:groups=core,resources=services,verbs=get;list;watch;create;update;patch;delete;
```

Comment:
you'd need this in dataplanerole_controller as well.

Reply:
I can do, but I have been testing this and with just this works perfectly. However, I am adding this to dataplanerole_controller just in case there is some case that I have not considered in my tests.

Reply:
Technically one annotation in one controller will then allow for all the other controllers to have the same permission, but for consistency's sake it's good to be explicit and annotate each controller to indicate exactly what permissions you expect them to have.